bin/index.js

```diff
@@ -37,10 +37,10 @@ if (OpenAPI) {
         clientName: params.name,
         useOptions: params.useOptions,
         useUnionTypes: params.useUnionTypes,
-        exportCore: JSON.parse(params.exportCore) === true,
-        exportServices: JSON.parse(params.exportServices) === true,
-        exportModels: JSON.parse(params.exportModels) === true,
-        exportSchemas: JSON.parse(params.exportSchemas) === true,
+        exportCore: params.exportCore,
+        exportServices: params.exportServices,
+        exportModels: params.exportModels,
+        exportSchemas: params.exportSchemas,
```

Comment:
@nicolas-chaulet I am only not sure about this change, but I couldn't quickly figure out how to test this file. Any ideas?